helms/legion-crds/templates/legion_v1alpha1_modeldeployment.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: modeldeployments.legion.legion-platform.org
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.state
+    name: State
+    type: string
+  - JSONPath: .spec.image
+    name: Model image
+    type: string
+  - JSONPath: .status.serviceURL
+    name: Service URL
+    type: string
+  group: legion.legion-platform.org
+  names:
+    kind: ModelDeployment
+    plural: modeldeployments
+    shortNames:
+    - md
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            annotations:
```

Comment:
I guess that allowing external control of annotations is unsecure, e.g. user can provide kube2iam annotation that may grant access to S3 bucket.